packs/csi_openstack_cinder/README.md

```diff
@@ -0,0 +1,29 @@
+# Openstack Cinder CSI
+
+This pack deploys the Openstack Cinder CSI container as a system job to all eligible Nomad Clients.
+
+See the Openstack Cinder CSI documentation for more information: https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/using-cinder-csi-plugin.md
+
+## Dependencies
+
+This pack requires the Nomad Client(s) be deployed on an Openstack VM that can have Cinder Volumes attached to it.
+
+## Container Privilege
+
+The Cinder Node task containers run as ***Privileged*** containers. The Cinder Controller tasks do not require privileged mode.
+
+## Variables
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| <a name="input_job_name"></a> [job\_name](#input\_job\_name) | The name to use as the job name which overrides using the pack name | `string` | `""` | no |
+| <a name="input_datacenters"></a> [datacenters](#input\_datacenters) | A list of datacenters in the region which are eligible for task placement | `list(string)` | <pre>[<br>  "dc1"<br>]</pre> | no |
+| <a name="input_region"></a> [region](#input\_region) | The region where the job should be placed | `string` | `"global"` | no |
+| <a name="input_constraints"></a> [constraints](#input\_constraints) | Constraints to apply to the entire job. | <pre>list(object({<br>    attribute = string<br>    operator  = string<br>    value     = string<br>  }))</pre> | <pre>[<br>  {<br>    "attribute": "${attr.platform.aws.placement.availability-zone}",<br>    "operator": "",<br>    "value": "nova"<br>  }<br>]</pre> | no |
+| <a name="input_job_restart_config"></a> [job\_restart\_config](#input\_job\_restart\_config) | n/a | <pre>object({<br>      attempts = number<br>      delay    = string<br>      mode     = string<br>      interval = string<br>    })</pre> | <pre>{<br>  "attempts": 5,<br>  "delay": "15s",<br>  "interval": "5m",<br>  "mode": "delay"<br>}</pre> | no |
+| <a name="input_cloud_conf_file"></a> [cloud\_conf\_file](#input\_cloud\_conf\_file) | Path to custom cloud.conf file to be mounted to the CSI containers. For reference, see https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md#global | `string` | `""` | no |
```

Comment:
Looks like this is required (see my comment on the template below)

Reply:
Yeah, I think I meant to delete the default declaration in this after copy-pasta'ng and just forgot. Removed the default and regenerated the table.